edgar-app/src/components/dashboardPages/documents/DocumentFirstFilter.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { HStack, Menu, MenuButton, MenuList, MenuItem, Icon } from '@chakra-ui/react';
+
+import UpArrowIcon from 'assets/icons/Arrow/UpArrowIcon';
+import DownArrowIcon from 'assets/icons/Arrow/DownArrowIcon';
+import CalendarIcon from 'assets/icons/CalendarIcon';
+
+interface DocumentFirstFilterProps {
+  onFilterChange: (value: string) => void;
+  sortOrder: string;
+}
+
+const DocumentFirstFilter: React.FC<DocumentFirstFilterProps> = ({ onFilterChange, sortOrder }) => {
+  const getIcon = (menuItemValue: string) => {
+    switch (menuItemValue) {
+      case 'asc':
+        return <Icon as={UpArrowIcon} w="12px" h="12px" />;
+      case 'desc':
+        return <Icon as={DownArrowIcon} w="12px" h="12px" />;
+      case 'newest':
+      case 'oldest':
+        return <Icon as={CalendarIcon} />;
+      default:
+        return null;
+    }
+  };

```

Comment:
No need to make this kind of function because you only call each icon one time into your code